x/tradeshield/genesis.go

```diff
@@ -17,6 +31,10 @@ func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
 	genesis := types.DefaultGenesis()
 	genesis.Params = k.GetParams(ctx)
 
+	genesis.PendingSpotOrderList = k.GetAllPendingSpotOrder(ctx)
+	genesis.PendingSpotOrderCount = k.GetPendingSpotOrderCount(ctx)
+	genesis.PendingPerpetualOrderList = k.GetAllPendingPerpetualOrder(ctx)
+	genesis.PendingPerpetualOrderCount = k.GetPendingPerpetualOrderCount(ctx)
```

Comment:
@amityadav0 because the module store was already registered as part of v0.45 we might need a migrator function in order to initiate the module states in Testnet.